AssignGamesManual.py

```diff
@@ -1,28 +1,28 @@
-from Objects import *
+from Objects import Round, Game, GetPlayerByPairingNumber, GetCombinedPreference, GetPlayerWeightForGame
 
-#This method can be used to create a manual pairing (in case the optimization doesn't work)
-def AssignGamesManual():
-    weighttotal = 0
-    weightlow = 1000
-    weightstotal = 0
-    weightslow = 1000
-    for r in Round.rounds:       
-        for m in r.matches:
-            p1 = GetPlayerByPairingNumber(m.p1nr)
-            p2 = GetPlayerByPairingNumber(m.p2nr)
-            m.prefs = {}
-            if p1.name == "BYE" or p2.name == "BYE":
+# This method can be used to create a manual pairing (in case the optimization doesn't work)
+
+
+def assign_games_manually():
+    for round in Round.rounds:
+        for match in round.matches:
```

Comment:
Qua naamgeving hebben functies altijd lowercase.
qua naamgeving is round, match duidelijker dan r, m, zeker als je veel code hebt.